ui/apps/platform/src/Containers/Clusters/cluster.helpers.test.js

```diff
@@ -338,14 +340,31 @@ describe('cluster helpers', () => {
             expect(received).toEqual(expected);
         });
 
-        it('should return "Manual upgrade required" if upgradeStatus -> upgradability is MANUAL_UPGRADE_REQUIRED', () => {
+        it(`should return "Version not managed by ${getProductBranding().reportName}" if upgradeStatus -> upgradability is MANUAL_UPGRADE_REQUIRED`, () => {
```

Comment:
If we know this message is for Sensor only, maybe we can say `Sensor is not managed by ${getProductBranding().reportName}.`